include/NovelRT/Graphics/RenderingService.h

```diff
@@ -23,6 +23,8 @@ namespace NovelRT::Graphics {
     ShaderProgram _texturedRectProgram;
     ShaderProgram _fontProgram;
 
+    RGBAConfig _framebufferColour;
+    
```

Comment:
This ordering of fields doesn't match the ordering initialised in the ctor I don't think? Could you check this please?